quizcon/templates/main/student_help.html

```diff
@@ -42,7 +42,7 @@ <h2>Try it out!</h2>
                                     <h3 class="text-center mt-2">Ready?</h3>
                                     {% if quiz.time %}
                                         <p class="text-center">
-                                            You will have <strong>{{quiz.time}} minutes</strong> to complete this quiz. <br />
+                                            You will have <strong>{{quiz.time}} {% if quiz.time == 1 %}minute{% else %}minutes{% endif %}</strong> to complete this quiz. <br />
```

Comment:
Check this Django template tag - https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#pluralize